src/styles/blocks/header.scss

```diff
@@ -0,0 +1,13 @@
+@import "../utils/mixins";
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  font-family: Roboto, sans-serif;
```

Comment:
set main font-family and font-size for whole page and just change it for classes where it is needed

Reply:
Done